doc/whatsnew/2.12.rst

```diff
@@ -12,6 +12,10 @@ Summary -- Release highlights
 New checkers
 ============
 
+* Pyreverse - add output in mermaid-js format
+
+* Fix ``install graphiz`` message which isn't needed for puml output format.
+
```

Comment:
This should be moved into the "Other changes" section. There also already is a bullet point for your last fix, so the "Fix ``install graphviz`` message ..." can be removed.